site/docs/v1/tech/apis/applications.adoc

```diff
@@ -162,6 +163,125 @@ include::docs/v1/tech/apis/_standard-put-response-codes.adoc[]
 
 include::docs/v1/tech/apis/_application-response-body.adoc[]
 
+== Search for Applications
+
+[NOTE.since]
+====
+This API has been available since 1.44.0
+====
+
+This API is used to search for Applications and may be called using the `GET` or `POST` HTTP methods. Examples of each are provided below. The `POST` method is provided to allow for a richer request object without worrying about exceeding the maximum length of a URL. Calling this API with either the `GET` or `POST` HTTP method will provide the same search results given the same query parameters.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Search for Applications using the given search and sort criteria
+[.endpoint]
+.URI
+--
+[method]#GET# [uri]#/api/application/search``?name=\{name\}&active=\{active\}&tenantId=\{tenantId\}&orderBy={orderBy}&startRow={startRow}``#
+--
+
+==== Request Parameters
+
+[field]#name# [type]#[String]# [optional]#Optional#::
+The string to search for in the Application name. This can contain wildcards using the asterisk or percent characters (* or %). This search parameter is case-insensitive.
+
+[field]#active# [type]#[Boolean]# [optional]#Optional#::
+Filters on the Application's `active` status. Can be omitted or set to `null` to retrieve both active and inactive Applications.
+
+[field]#tenantId# [type]#[UUID]# [optional]#Optional#::
+Restricts the results to Applications belonging to the given Tenant. This parameter will be overridden if the request contains an `X-FusionAuth-TenantId` header, or if the supplied API key is scoped to a specific Tenant.
+
+[field]#numberOfResults# [type]#[Integer]# [optional]#Optional# [default]#defaults to `25`#::
+The number of results to return from the search.
+
+[field]#orderBy# [type]#[String]# [optional]#Optional# [default]#defaults to `insertInstant DESC`#::
+The database column to order the search results on plus the order direction.
++
+The columns you can use for this are:
++
+ * `id` - the UUID of the Application
+ * `insertInstant` - the link:/docs/v1/tech/reference/data-types#instants[instant] when the Application was created
+ * `name` - the Application name
+ * `tenant` - the name of the Tenant in which the Application belongs
++
+For example, to order the results by the insert instant in a descending order, the value would be provided as `insertInstant DESC`. The final string is optional can be set to `ASC` or `DESC`. If omitted, the default sort order is `ASC`.
+
+[field]#startRow# [type]#[Integer]# [optional]#Optional# [default]#defaults to `0`#::
+The offset row to return results from. If the search has 200 records in it and this is 50, it starts with row 50.
```

Comment:
It may be worth doing a quick audit to ensure we are consistely describing this parameter across all of our search APIs. 

Here is the text from the Entity Types and Login Records APIs.

> The offset into the total results. In order to paginate the results, increment this value by the [field]#numberOfResults# for subsequent requests.


While it is sort of obvious - I kind of like adding the `increment this value by the [field]#numberOfResults# for subsequent requests.` to help developers know how to paginate.

Reply:
Combining them both may be a good option, like: 
```The offset row to return results from. If the search has 200 records in it and this is 50, it starts with row 50. In order to paginate the results, increment this value by the [field]#numberOfResults# for subsequent requests.```

Reply:
I think we have a mix.. but in general it doesn't read very well to end a sentence with `from`. 

Here is how we say it other places in the API docs which I think reads better, and may be more grammatically correct. But I know we have some grammar nerds.. so I'll defer there. 

> The offset into the total results.

&nbsp;

For the second part of the description, this sounds like an example, so I would prefer to use our "For example..." syntax. Here is a suggestion:

```
[field]#startRow# [type]#[Integer]# [optional]#Optional# [default]#defaults to `0`#::
The offset into the total results. In order to paginate the results, increment this value by the [field]#numberOfResults# for subsequent requests.
+
For example, if the total search results are greater than the page size designated by [field]#numberOfResults#,  to request the second page of results, assuming the default page size, set this value to `25` to retrieve results `26-50`.
```